src/lib/Layout/PanelSideBarLayout/PanelSideBar/PanelSideBarItem.tsx

```diff
@@ -31,12 +33,22 @@ const PanelSideBarItem = (props: PanelSideBarItemProps) => {
         style={{ paddingLeft: depth ? `${depth + 1}rem` : undefined }}
       >
         {hasitem ? (
-          <div>
-            <a role="button" className={classNames("nav-link", { "dropdown-toggle": hasitem })} onClick={() => setIsOpen(!isOpen)}>
```

Comment:
Was not simpler to render the linkrendered before the <a> ifcollapseIconOnly excluding its content, otherwise keeping the version like it is now?

Reply:
@manni497 Fixed